test/games/strategy/triplea/delegate/GameDataTestUtil.java

```diff
@@ -80,6 +80,27 @@ public static UnitType carrier(final GameData data) {
     return unitType(Constants.UNIT_TYPE_CARRIER, data);
   }
 
+  // Some units hard coded here rather than placed in Constants at the request of community members Ref: Pull Request 1074
```

Comment:
You must have been using some really old monitor then. Like 10 years old. 1920x1280 capable monitors have been around for a pretty long time.

Console windows? You mean like the Wyse terminal I was referring to above? I doubt much Java is worked on that way and I think it adds strongly negative value to cater to it.